src/workerd/jsg/jsg.h

```diff
@@ -229,6 +230,12 @@ namespace workerd::jsg {
     registry.template registerStaticMethod<NAME, decltype(Self::method), &Self::method>();         \
   } while (false)
 
+#define JSG_RUST_STATIC_METHOD_NAMED(name, method)                                                 \
```

Comment:
This needs documentation around it. Specifically, it is different from the other `JSG_STATIC_*` methods which register on the class itself rathe than on the prototype... e.g. `JSG_RESOURCE_TYPE(Foo) { JSG_STATIC_METHOD(bar); }` would map to `Foo.bar()`, whereas these are on the prototype if I'm understanding correctly.

The documentation should also indicate how `this` is handled, if at all. Basically just give some more context around correct usage / expectations.

Reply:
I've renamed this method to JSG_RUST_METHOD_NAMED. Which is a better name IMHO.